doc/gettingstarted.md

```diff
@@ -21,32 +23,31 @@ In this guide we will get everything up and running on your local machine (on `l
 
 ## Dependencies 
 
-#### Certificates
+### CLI and Stack
+
+For **stable** versions, see the [Releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on Github.
```

Comment:
```suggestion
Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
```

Reply:
How about mentioning common commands for Mac (`brew ...`) and Ubuntu (`apt ...`)?

Reply:
I think it would be appropriate to put those instructions in the descriptions of the releases.

Reply:
I'm not so sure. It would be in every release, and it's what people do but it misses the context of getting started. That's here.

Also, I'd like to avoid that people have to go to other pages in order to pass the getting started. A self-containing getting started would be ideal for me.

Reply:
But then where does it end? Is the Getting Started going to tell how to install the binaries using Apt? Brew? Yum? Nix? Chocolatey? Which files to download? IMO that belongs on a "Downloads" page that the Getting Started will refer to.

If we combine that with the Releases, we can point people to the exact binary versions instead of having to specify that in some other way.

Reply:
Yeah it's a sliding edge, I agree. OK fine to put it in the releases and link there then.

Reply:
@adriansmares so my suggestion here as first comment is still valid